main/main.cpp

```diff
@@ -830,9 +831,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	GLOBAL_DEF("rendering/quality/intended_usage/framebuffer_allocation", 2);
 	GLOBAL_DEF("rendering/quality/intended_usage/framebuffer_allocation.mobile", 3);
 
-	if (editor) {
-		OS::get_singleton()->_allow_hidpi = true; //editors always in hidpi
+	if (editor || project_manager_request) {
```

Comment:
This won't work `Main::Start` is called after `Main::Setup` and `project_manager_request` will be always false here. You should move all `project_manager_request` related code (lines 1190-1238) from Start to Setup as well.

Reply:
It's too late, `_allow_hidpi` should be set before window is created (it is checked `OS::initialize`), this can be changed on macOS, but IIRC on Windows `SetProcessDpiAwareness` can be called only once, before any GUI api calls.

Reply:
Ok, so @Calinou you should add this in `Main::setup` in the big while loop (after the check for `-e`, `--editor`):
```
		} else if (I->get() == "-p" || I->get() == "--project-manager") { // starts project manager

project_manager = true;
```
Then it should work™

Reply:
+ a style fix needed (either keep both parts on the same line, or do the clang-format changes as proposed, as you wish): https://travis-ci.org/godotengine/godot/jobs/341071551

Reply:
Eventually I'll refactor `main.cpp` to do better argument parsing upfront, and then streamline the `setup` and `start` stuff, but for the time being this fix should do the trick.